DynamoLeagueBlazor/Tests/UITestBase.cs

```diff
@@ -1,5 +1,63 @@
-﻿namespace DynamoLeagueBlazor.Tests;
+﻿using Microsoft.Extensions.DependencyInjection;
+using MockHttp;
+using MockHttp.Json;
+using MockHttp.Json.Newtonsoft;
+using MockHttp.Language;
+using Moq;
+using MudBlazor;
+using MudBlazor.Services;
 
-internal class UITestBase
+namespace DynamoLeagueBlazor.Tests;
+
```

Comment:
Remove this (for now) until we can separate between UI and Integration test fixtures